polyfill/lib/civildate.mjs

```diff
@@ -13,6 +13,7 @@ export class CivilDate {
     const { year, month, day } = plus({}, { years, months, days });
     this[DATA] = { year, month, day };
   }
+  get [Symbol.toStringTag] () { return 'CivilDate'; }
```

Comment:
these shouldn’t be getters (unless they’re going to brand check before returning the string, which I’d be super in favor of)